charts/camunda-platform-alpha/templates/identity/configmap.yaml

```diff
@@ -213,6 +223,43 @@ data:
     {{- if .Values.global.identity.auth.enabled }}
     {{- if eq .Values.global.identity.auth.type "KEYCLOAK"}}
     keycloak:
+      url: {{ include "identity.keycloak.url" . | quote }}
+      setup:
+        user: {{ include "identity.keycloak.authAdminUser" . | quote }}
+        password: ${VALUES_KEYCLOAK_SETUP_PASSWORD:}
+      init:
```

Comment:
It should be as it's a 1:1 rename of the env vars like `KEYCLOAK_INIT_OPERATE_SECRET`.
here is an example:
https://github.com/camunda-cloud/identity/blob/a61fbd804ec663512be362c36ea4745b78a4a968/management-api/src/test/resources/preset-initialization-test.yaml#L59

Also, @Ben-Sheppard could confirm.

Reply:
It should be as it's a 1:1 rename of the env vars like `KEYCLOAK_INIT_OPERATE_SECRET`.
here is an example:
https://github.com/camunda-cloud/identity/blob/a61fbd804ec663512be362c36ea4745b78a4a968/management-api/src/test/resources/preset-initialization-test.yaml#L59

Also, @Ben-Sheppard could confirm.

Reply:
I observed my Applications  did get the proper passwords from my existingSecret.  though I wonder if thats because of the settings below or because of lines like  

https://github.com/camunda/camunda-platform-helm/blob/b308d2d7a963d4cf43a5991dbb93364a854c2a9b/charts/camunda-platform-alpha/templates/identity/configmap.yaml#L145

Reply:
I got your comment late @aabouzaid , after posting my previous 2 messages.

Reply:
Thats correct, the test is a good indicator here of the yaml variation here :) - they map to this variable that is used in the Identity initialisation flows https://github.com/camunda-cloud/identity/blob/main/management-api/src/main/java/io/camunda/identity/impl/keycloak/config/record/KeycloakCommon.java#L28

Reply:
@Ben-Sheppard Thanks for your feedback :raised_hands: